components/Header/index.js

```diff
@@ -9,4 +9,30 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+function Header() {
+  // Ceate elements
+  const header = document.createElement('div');
+  const date = document.createElement('span');
+  const mainHeading = document.createElement('h1');
+  const temp = document.createElement('span');
+
+  // Add classes
+  header.classList.add('header');
+  date.classList.add('date');
+  temp.classList.add('temp');
+
+  // Structure
+  header.appendChild(date);
+  header.appendChild(mainHeading);
+  header.appendChild(temp);
+
+  // Add value
+  date.textContent = `SMARCH 28, 2019`;
```

Comment:
you could have up dated this to read the date using the date() that is built into javaScript